File: print-manager-dotnet-core-cs/InputPagesExample.cs


using ceTe.DynamicPDF.Printing;

namespace print_manager_dotnet_core_cs
{
    internal class InputPagesExample
    {
        public static void Run()
        {

            PrintJob printJob = new PrintJob(Printer.Default);
            InputPdf inputPdf = new InputPdf(Util.GetPath("Resources/DocumentA.pdf"));
            InputPdfPage firstPage = inputPdf.Pages[0];
            InputPdfPage fourthPage = inputPdf.Pages[3];
            printJob.Pages.Add(firstPage);
            printJob.Pages.Add(fourthPage);
            printJob.Print();
        }
    }
}

File: print-manager-dotnet-core-vb/InputPagesExample.vb
Imports ceTe.DynamicPDF.Printing


Friend Class InputPagesExample

    Public Shared Sub Run()

        Dim printJob As New PrintJob(Printer.Default)
        Dim inputPdf As New InputPdf(Util.GetPath("Resources/DocumentA.pdf"))

        Dim firstPage As InputPdfPage = inputPdf.Pages(0)
        Dim fourthPage As InputPdfPage = inputPdf.Pages(3)

        printJob.Pages.Add(firstPage)
        printJob.Pages.Add(fourthPage)

        printJob.Print()

    End Sub

End Class
